demo/solid/src/index.js

```diff
@@ -0,0 +1 @@
+export * from "./main.cljs"
```

Comment:
Have you looked into whether this index.js workaround is still necessary with the various improvements you've made to the plugin lately?

Reply:
It is still necessary at the moment, the only way we can remove this is by adding https://vitejs.dev/guide/api-plugin#transformindexhtml rewriting the `cljs` script tag

Reply:
I will open an issue to track this

Reply:
Issue created: #17